src/SignalR/clients/ts/FunctionalTests/scripts/run-tests.ts

```diff
@@ -99,7 +99,7 @@ function waitForMatches(command: string, process: ChildProcess, regex: RegExp, m
     });
 }
 
-let configuration = "Debug";
+let configuration = process.env['npm_config_configuration'] || "Debug";
```

Comment:
This is just so that it can be picked up from the CLI when running `npm run test -- --configuration=Release`